Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -152,26 +158,28 @@ static MTLStoreAction storeAction(WGPUStoreOp storeOp)
             attachment.clearColor.b,
             attachment.clearColor.a);
 
+        addTextureStorageIfNeeded(fromAPI(attachment.view), attachment.resolveTarget ? WGPUStoreOp_Discard : attachment.storeOp, attachment.loadOp);
         mtlAttachment.texture = fromAPI(attachment.view).texture();
         mtlAttachment.level = 0;
         mtlAttachment.slice = 0;
         mtlAttachment.depthPlane = 0;
         mtlAttachment.loadAction = loadAction(attachment.loadOp);
-        mtlAttachment.storeAction = attachment.resolveTarget ? MTLStoreActionStoreAndMultisampleResolve : storeAction(attachment.storeOp);
 
         if (attachment.resolveTarget) {
-            mtlDescriptor.colorAttachments[i].resolveTexture = fromAPI(attachment.resolveTarget).texture();
+            mtlAttachment.resolveTexture = fromAPI(attachment.resolveTarget).texture();
```

Comment:
Is this correct? This seems like the sort of thing that should be covered by a test. If nothing in the CTS covers this, can we write a test and upstream it to the CTS?

Reply:
`mtlAttachment` and `mtlDescriptor.colorAttachments[i]` are the same, there is no logical change here